packages/component-elife-app/server/authsomeMode.js

```diff
@@ -0,0 +1,29 @@
+const BluebirdPromise = require('bluebird')
+
+// Policy type: (user: string, operation: string, object: any, context?: any) => boolean
+const authsomePolicies = {
```

Comment:
We need to have some sort of discussion about how we organise this list of policies: where a policy for a specific use case will reside; conventions for how we write policies and is there a better, less repetitive, safer way of doing it.

Reply:
we could use constants ?

Reply:
I just really don't like referring to the policies with strings - I feel like it's a really fragile way of referring to an important part of our system logic. I think it might be worth investing some time to build a really flexible policy system but it would probably be quite ~difficult~ interesting to do well.